docs/examples/hypervisors/kvm/terraform.tfvars

```diff
@@ -0,0 +1,40 @@
+###############################
+#### Environment variables ####
+###############################
+tenancy_ocid="ocid1.tenancy.oc1..abcabcababccbcbaabcabababcbabababcbababcbabababacbcabcabcabac4uq"
+compartment_ocid="ocid1.tenancy.oc1..abcabcababccbcbaabcabababcbabababcbababcbabababacbcabcabcabac4uq"
+user_ocid="ocid1.user.oc1..aaaaaaaaubbbbbbbbbbccccccccccdddddd64fizdmbjblpq"
+fingerprint="aa:bb:cc:dd:5e:ff:ee:47:9b:3c:5f:c5:69:02:1b:aa"
+ssh_private_key_path="~/.ssh/id_rsa"
+ssh_public_key_path="~/.ssh/id_rsa.pub"
+region="us-phoenix-1"
+#region="us-ashburn-1"
+
+#######################################
+#### Instance definition variables ####
+#######################################
+
+50GB = "51200"
```

Comment:
I'm not seeing where this is used, and when I do tf plan it's complaining about this.

Reply:
`terraform.tfvars` is one of the recommended approaches to specify [input variables](https://www.terraform.io/intro/getting-started/variables.html) to templates on terraform. Terraform automatically loads this file and populates all the variables. Make sure you run terraform in the following order: get, init and plan . `Get` loads the module definition files where all these variables are specified.

Reply:
It's look like a Terraform bug. It's complaining about the 50GB variable, it's identifying the variable as a NUMBER instead of STRING. I will modify the code as a workaround and open an issue on Terraform.

Reply:
I just removed the variable from the files and documentation. It was not used.